### PR TITLE
Add an option to create dot model in different directions

### PR DESCRIPTION
**Pull request by eroniki**
This commit adds an optional argument to functions plot() and model_to_dot() specifying the direction of the dot object

### CODE REVIEW

**Comment by deepersystems** [comment]
https://github.com/deepersystems/consolenet

**Comment by fchollet** [comment]
That's reasonable; please fix the merge conflicts and we'll merge it. Sorry for the delay.

**Comment by eroniki** [comment]
I believe it is done!

**Comment by fchollet** [comment]
PEP8 issue: https://travis-ci.org/fchollet/keras/jobs/219284824

**Comment by fchollet** [comment]
Another PEP8 issue: https://travis-ci.org/fchollet/keras/jobs/219492699

File: keras/utils/vis_utils.py [hunk 1]
```diff
@@ -18,7 +18,10 @@ def _check_pydot():
...
 
 
-def model_to_dot(model, show_shapes=False, show_layer_names=True):
+def model_to_dot(model,
+                 show_shapes=False,
+                 show_layer_names=True,
+                 rankdir='TB'):
     """Converts a Keras model to dot format.
 
     # Arguments
```
Comment (Dref360) [inline keras/utils/vis_utils.py]: No Doc for the new args?

Comment (fchollet) [inline keras/utils/vis_utils.py]: Ideally we should write a test to automatically check docstring existence and correctness. This is on our to-do list: https://github.com/fchollet/keras/projects/1

**Comment by fchollet** [comment]
Any update? In particular with respect to the failing PEP8 test? Docstring update?

**Comment by eroniki** [comment]
@fchollet I forgot to fix the code. I will inform you when it is done, hopefully by the afternoon. Thanks for reminding me!

**Comment by fchollet** [comment]
LGTM, thanks

**Comment by eroniki** [comment]
@fchollet Done!

**Comment by Dref360** [comment]
LGTM

**Pull request merged**